packages/core/src/node/debug.ts

```diff
@@ -22,7 +22,7 @@ function isInDebugMode(): boolean {
     }
     if (process && process.execArgv) {
         return process.execArgv.some(arg =>
-            /^--debug=?/.test(arg) || /^--debug-brk=?/.test(arg)
+            /^--(debug|inspect)(-brk)?=?/.test(arg)
```

Comment:
Should not it be already by `typeof v8debug === 'object'`?

Reply:
That's what I thought at the beginning, but for some reason I am unable to have this `v8debug` global var to exists in my environment :(

```
$ node --inspect
Debugger listening on ws://127.0.0.1:9229/03901ae0-32f9-44d0-a8ba-967e3aadfd9e
For help see https://nodejs.org/en/docs/inspector
> v8debug
ReferenceError: v8debug is not defined
> 
```